samples/matter/template/suit_templates/app_envelope.yaml.jinja2

```diff
@@ -0,0 +1,148 @@
+{%- set mpi_application_vendor_name = application['config']['CONFIG_SUIT_MPI_APP_LOCAL_1_VENDOR_NAME']|default('nordicsemi.com') %}
```

Comment:
Do this .jinja2 files require documentation? Or they are not something that will be modified by the user, so we can just put them in the sample dir and forget about them?

Reply:
We will continue to work on that after the NCS 2.7.0 release, for now, they are generated automatically there.